src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "properties": [
+    {
+      "name": "encryption.secret",
+      "type": "java.lang.String",
+      "description": "The key that should be used to encrypt and decrypt unsubscription links."
+    },
+    {
+      "name": "encryption.salt",
+      "type": "java.lang.String",
+      "description": "Some salt to be added to the initial vector."
```

Comment:
What is salt? 😅

Reply:
Adds spice to the enryption 😄 

https://en.wikipedia.org/wiki/Salt_(cryptography)